index.bs

```diff
@@ -542,6 +542,19 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 			* Custom format [=string/starts with=] `"web "`("web" followed by U+0020 SPACE) prefix
 				and suffix (after stripping out `"web "`) passes the [=parsing a MIME type=] check.
 
+	<h3 id="unsanitized-data-types-x"><dfn>Unsanitized data types</dfn></h3>
+
+		The implementation MAY recognize the native OS clipboard format description
+		for the following data types, to be able to populate the
+		{{ClipboardItem}} with the correct description for paste events, and
+		set the correct data format on the OS clipboard in response to copy and cut
+		events.
+
+			These data types must not be sanitized and should be exposed by UAs
```

Comment:
Is it right to say "must not be sanitized"? We should only not sanitize if these types are passed as part of the unsanitized option.

Reply:
The intention is to not sanitize any formats that are in the `Unsanitized data types` list. If the web authors specify any other MIME types, then we will reject that MIME type during `read`.